src/button/Button.d.ts

```diff
@@ -21,9 +22,17 @@ export type ButtonProps = {
     fluid?: boolean,
     circular?: boolean,
     textOverflow?: React.ComponentClass<object>, // todo(typo) when translate text overflow, update here its props types
-    style?: React.CSSProperties,
+    style?: CSSProperties,
     className?: string
   }
 
```

Comment:
what about StatefulButtonProps?
they should be `ButtonProps & { baseState?: BaseState, stableSuccess?: boolean, timerMillis?: number }`